source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,119 @@
+---
+title: oVirt Software Defined Networking, The OVN Network Provider
+author: mmirecki
+date: 2016-10-03 14:21:00 UTC
+tags: community, howto, blog, network
+comments: true
+published: true
+---
+
+oVirt offers not only its own internal networking, but also an API for external network providers. This API enables using external network management software inside environments managed by oVirt and takes advantage of their extended capabilities. 
+One of such solutions is OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to OVS.
+
+OVN enables support for virtual networks abstraction by adding native OVS support for virtual L2 and L3 overlays.
```

Comment:
please add:
In the near future (put a link to Lance's IPAM patch) it would make it possible to run oVirt-hosted VMs dynamically, without on-site DHCP server.